__tests__/app.test.js

```diff
@@ -0,0 +1,36 @@
+const app = require("../app");
+const connection = require("../db/connection");
+const request = require("supertest");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index");
+
+beforeEach(() => seed(testData));
+
+describe("GET /api/topics", () => {
+  it("should return an array", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((result) => {
+        let { body } = result;
+        expect(Array.isArray(body)).toBe(true);
+      });
+  });
+  it("should return an array of objects with the correct keys", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((result) => {
+        let { body } = result;
+        body.forEach((topic) => {
```

Comment:
Whenever we test with a forEach, we should also check the length of the array. If that array happens to be empty, we want to know that